There are a few failed sanity tests which you should fix as well:
```
lib/ansible/plugins/connection/docker.py:78:1: W293: blank line contains whitespace
lib/ansible/plugins/connection/docker.py:78:0: trailing-whitespace: Trailing whitespace
lib/ansible/plugins/connection/docker.py:82:1: W293: blank line contains whitespace
lib/ansible/plugins/connection/docker.py:82:0: trailing-whitespace: Trailing whitespace
```
(I can't see any whitespace in GitHub files view, it could potentially be a Windows line ending instead of a Unix line ending.)

--- a/changelogs/fragments/67832-run_powershell_modules_on_windows_containers.yml
+++ b/changelogs/fragments/67832-run_powershell_modules_on_windows_containers.yml
@@ -1,0 +1,2 @@
+minor_changes:
+  - docker connection plugin - run Powershell modules on Windows containers.

--- a/lib/ansible/plugins/connection/docker.py
+++ b/lib/ansible/plugins/connection/docker.py
@@ -75,11 +75,11 @@ class Connection(ConnectionBase):
         # error it probably means that docker on their system is only
         # configured to be connected to by root and they are not running as
         # root.
-        
+
         # Windows uses Powershell modules
         if getattr(self._shell, "_IS_WINDOWS", False):
             self.module_implementation_preferences = ('.ps1', '.exe', '')
-            
+
         if 'docker_command' in kwargs:
             self.docker_cmd = kwargs['docker_command']
         else:

